object_storage_api/stores/attachment.py

```diff
@@ -16,6 +17,26 @@ class AttachmentStore:
     Store for managing attachments in an S3 object store.
     """
 
+    def create_presigned_get(self, attachment: AttachmentOut) -> str:
+        """
+        Generate a presigned url to share an S3 object.
+
+        :param attachment: `AttachmentOut` model of the attachment
+        :return: Presigned url to get the attachment.
+        """
+        logger.info("Generating presigned url to get attachment from object storage")
```

Comment:
```suggestion
        logger.info("Generating presigned url to get attachment from object storage")
```
Might be worth including the object key in the logging message (similar to the image store message) to help with debugging in the future.